mlos_bench/mlos_bench/service/base_service.py

```diff
@@ -61,19 +62,23 @@ def __init__(self, config: dict = None, parent=None):
         """
         self.config = config or {}
         self._parent = parent
-        self._services = {}
+        self._services: Dict[str, Callable] = {}
 
         if parent:
             self.register(parent.export())
 
+        self._config_loader_service: SupportsConfigLoading
```

Comment:
`ConfigPersistenceService` is so often expected to be in the base of the Service somewhere that I thought it might be useful to just give an easy reference to the typed version of it for others to use.

Reply:
agree. the problem is that it has to be instantiated before any other services. Let's think about it and do it properly in the subsequent PRs